.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "documentation"]
```

Comment:
what is this one?

Reply:
Oh, that was my ill-fated attempt to get this working with sub-modules, thanks for catching this, I thought I'd deleted it.